packages/components/pages/f-checkout/stories/api/responses/getBasket/getBasket.js

```diff
@@ -9,33 +9,31 @@ const restriction = {
     Type: 'Alcohol'
 };
 
-function buildPrompts (issues) {
-    return {
-        SpendMore: null,
-        DiscountApplied: {
-            Amount: 0
+const buildPrompts = issues => ({
```

Comment:
this is fine, but I think next time it might be simpler to raise a separate PR straight after that contains purely linting fixes 😃